wiki/Tournaments/Common_seeding_methods/en.md

```diff
@@ -0,0 +1,181 @@
+- When scores are more clustered near the mean and more spread out for outliers – more precisely, when the distribution of scores is close to a bell curve – Z-percentile will favor teams similarly to Placement, other than the fact that very close scores will receive similar points (for example Teams 2 through 4 above). But if scores are more uniformly distributed, Z-percentile instead favors teams more similarly to Percent Max.
+- If the highest score of 950,000 were instead much higher (like 1,200,000), seeding under Placement would remain the same, seeding under Percent Max would heavily deflate all other teams, and seeding under Z-percentile would be altered slightly (most notably in shifting the average and thus the range of scores in which improvements are most important).
+
+Finally, a side-by-side comparison of all seeding methods on this page using the [osu! World Cup 2024](/wiki/Tournaments/OWC/2024) Qualifiers data can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/e/2PACX-1vSNjTR6oXcjyCEaNEu-h0EabC8ZSYUMVhjEUYvZsfU78JXOrUpoOc_lIDinRe2gWEDbDCkTdvr1VCn9/pubhtml).
+
+### Further analysis
+
+Here are some other features and comparisons of seeding methods to keep in mind, especially for tournaments using methods not listed on this page.
+
+- Some seeding methods are not very sensitive overall to individual teams' scores, while others may be changed more dramatically.
+  - For example, Sum of Placements yields the same placements no matter how large the differences between team scores are, but a particularly high or low score in Percent Difference may cause all other `map points` to be unusually close together. This is particularly important for modes like osu!taiko or osu!mania where scores are typically very close to the maximum possible; historically, some tournaments such as [SOFT](/wiki/Tournaments/SOFT) have instead used a "piecewise-linear" Percent Difference method by grouping scores by quartile.
+  - For another kind of example, the Z-sum and Z-percentile seeding methods have the feature that very close scores will receive very close point values for seeding (so if there is a cluster of many nearby scores, the highest and lowest scores of those clusters will be considered similarly in seeding), which is behaviour that does not occur for Sum of Placements.
+- Some seeding methods benefit specialists, while others benefit consistent performances. For example, Zipf's Law will give a large boost in seeding to teams with the highest or near-highest rank on a map compared to Sum of Placements. And because of the shape of the normal distribution, Z-sum will give a large boost for extremely high scores (and a large reduction for extremely low scores) compared to Z-percentile; thus, Z-percentile tends to favour all-rounders over specialists compared to Z-sum.
```

Comment:
```suggestion
- Some seeding methods benefit specialists, while others benefit consistent performances. For example, Zipf's law will give a large boost in seeding to teams with the highest or near-highest rank on a map compared to sum of placements. Similarly, because of the shape of the normal distribution, Z-sum will give a large boost for extremely high scores (and a large reduction for extremely low scores) compared to Z-percentile. Z-percentile therefore tends to favour all-rounders over specialists compared to Z-sum.
```